client/i18n/locales/english/translations.json

```diff
@@ -429,6 +429,8 @@
     "answered-mcq": "You have unanswered questions and/or incorrect answers.",
     "explanation": "Explanation",
     "transcript": "Read a transcript of this video",
```

Comment:
Is this still needed?

Reply:
No, it's not. For some reason I kinda like to keep them around in case we want to use it again, but there's really no need to.